src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -187,7 +193,8 @@ public SlaveTemplate(String ami, String zone, SpotConfiguration spotConfig, Stri
             String instanceCapStr, String iamInstanceProfile, boolean deleteRootOnTermination,
             boolean useEphemeralDevices, boolean useDedicatedTenancy, String launchTimeoutStr, boolean associatePublicIp,
             String customDeviceMapping, boolean connectBySSHProcess, boolean monitoring,
-            boolean t2Unlimited, ConnectionStrategy connectionStrategy, int maxTotalUses) {
+            boolean t2Unlimited, ConnectionStrategy connectionStrategy, int maxTotalUses,
```

Comment:
This breaks backward compatibility. You can either add this as a setter method with `@DataBoundSetter`, or you'll have to deprecate the old constructor and add this new one just like the dozens below.

Reply:
Done